demo/.storybook/main.js

```diff
@@ -12,13 +12,4 @@ module.exports = {
     name: '@storybook/react-webpack5',
     options: {},
   },
-  swc: () => ({
-    jsc: {
-      transform: {
-        react: {
-          runtime: 'automatic',
-        },
-      },
-    },
-  }),
```

Comment:
Not need with the latest https://github.com/storybookjs/addon-webpack5-compiler-swc

Reply:
Seeing https://storybook.js.org/docs/api/main-config/main-config-swc doesn't say that this particular stuff can be removed. Does the package have defaults that do this now?